node.c

```diff
@@ -1145,8 +1145,12 @@ init_node_buffer_list(node_buffer_list_t * nb, node_buffer_elem_t *head)
 static node_buffer_t *
 rb_node_buffer_new(void)
 {
-    size_t bucket_size = offsetof(node_buffer_elem_t, buf) + NODE_BUF_DEFAULT_LEN * sizeof(NODE);
-    node_buffer_t *nb = xmalloc(sizeof(node_buffer_t) + (bucket_size * 2));
+    const size_t bucket_size = offsetof(node_buffer_elem_t, buf) + NODE_BUF_DEFAULT_LEN * sizeof(NODE);
+    const size_t alloc_size = sizeof(node_buffer_t) + (bucket_size * 2);
+    STATIC_ASSERT(
+        "integer overflow",
```

Comment:
This argument is a part of an identifier, not a string literal.

Reply:
Thanks, I was not aware of compilers without _Static_assert.